lib/services/plugins/npm-plugins-service-base.ts

```diff
@@ -349,9 +349,16 @@ export abstract class NpmPluginsServiceBase implements IPluginsService {
 				suppressMessage: !suppressMessage
 			};
 
-			options.originalPluginDirectory = pluginLocalPathExists && pluginLocalPath;
+			let pluginId = pluginIdentifier;
```

Comment:
Why do you need the new variable. As far as I see, `pluginIdentifier` is not used anywhere else, so you can just set it to `pluginLocalPath` in the `if` body